contracts/FlowIDTableStaking.cdc

```diff
@@ -1512,9 +1513,6 @@ pub contract FlowIDTableStaking {
                 nodeRecord.tokensRequestedToUnstake = 0.0
             }
 
-            // Save the updated moves pending list
-            FlowIDTableStaking.storeNewMovesPending(movesPendingList)
```

Comment:
We don't need this anymore, because modifications to a borrowed object are always persisted back to storage? Just checking :)

Reply:
yes, that is correct